### ISSUE TITLE
Change coder-table output name/location

### ISSUE THREAD

**Issue by davidskalinder** [davidskalinder/mpeds-coder#62]
Hopefully this will get more seriously overhauled soon, but for now it'd be nice to have the time in the coder-table filename in case of multiple runs in a day.

@alexhanna, I think some of your cron jobs run with this file?  So let me know if changing the filename will be the thing that creates a permanent rift with the upstream project...

**Comment by davidskalinder** [comment]
Looping in @olderwoman non-diegetically

**Comment by davidskalinder** [comment]
All right, this is changed in branch codertable_filename_time, merged into branch testing and live in the testing deployment.

The new filenames look like `/var/www/bpp_development/coder-table_2020-03-16_142232.csv`.  I think it makes sense to also change the article-level export filenames similarly, but I'll log that in #56.

**Comment by davidskalinder** [comment]
Changed filenames to look like `bpp_coder_article_annotation_2020-03-16_145339.csv`.

@olderwoman, this might require an update to your scripts?  I change the name manually when I create the `_alt` files anyway, so if it needs a change it should be easy to tweak that way if necessary...

**Comment by davidskalinder** [comment]
@olderwoman, I think it's worthwhile to test this in the testing deployment before merging it into production?  But let me know it you'd prefer I take it into production right away and we can fix it there if necessary...

**Comment by davidskalinder** [comment]
Now that some other issues are creating some exports, I created an exports directory and changed the paths so that coder-table files go to the exports directory.  Changed the branch name to `codertable_output_location` and merged it into `testing`.

**Comment by davidskalinder** [comment]
Now live in the testing deployment.  Looks good to me, but I'll wait for a second set of eyes before closing this.

**Comment by davidskalinder** [comment]
Moving an email discussion here so we can find it later...

So, @olderwoman sez:
> Um the ADVANTAGE to having identical timestamps on two files that have to be merged is that you need to declare the timestamp only once, instead of twice and can stamp all the derived files with the common timestamp.  Why would you not want to do this? I don’t care at what millisecond the file was created, I want to know exactly which files go with each other.

Ah, got it, that makes sense.  I didn't know you were using exact timestamp matches to find the files.  Since at the moment the two files are created by two totally separate processes, there's no way to automate this (though of course in the short term I could manually rename one or both of the files when I create them).

In the medium term, the idea of using matching timestamps makes me nervous; after all, the files don't really match perfectly -- one has to be created before the other, and we can't guarantee there are no changes in between.

But as I understand it the problem is that it's rather hard to keep track of the files?  I was thinking of improving this situation by creating a subfolder within `exports` called `old` and moving everything except the most recent set of exports there.  Would that solve the problem?

(In the long term, there shouldn't be two files -- just one output of a join that happens in the database, which will guarantee that everything comes from a single state of the data.)

And from another part of the exchange; first from me:
>Yeah, so the testing interface should write to the testing folder and the production interface should write to the production folder.  At the moment nothing has changed in production, so everything should happen in the testing interface only.

... and the reply from @olderwoman:
> IN FACT when I generated coder table it wrote to THREE locations, or at least two or them.

So, if so, I can't find them?  Here's all the coder-table files that have been created in the three deployments whose name includes any date in April:

```
skalinder@sheriff:/var/www$ ls -l bpp/coder-table*2020-04*
-rw-r--r-- 1 www www 8155714 Apr  3 19:13 bpp/coder-table_2020-04-03.csv

skalinder@sheriff:/var/www$ ls -l bpp/exports/coder-table*2020-04*
ls: cannot access 'bpp/exports/coder-table*2020-04*': No such file or directory

skalinder@sheriff:/var/www$ ls -l bpp_testing/coder-table*2020-04*
ls: cannot access 'bpp_testing/coder-table*2020-04*': No such file or directory

skalinder@sheriff:/var/www$ ls -l bpp_testing/exports/coder-table*2020-04*
-rw-r--r-- 1 www www 1666912 Apr  3 17:56 bpp_testing/exports/coder-table_2020-04-03_175651.csv

skalinder@sheriff:/var/www$ ls -l bpp_development/coder-table*2020-04*
ls: cannot access 'bpp_development/coder-table*2020-04*': No such file or directory

skalinder@sheriff:/var/www$ ls -l bpp_development/exports/coder-table*2020-04*
-rw-r--r-- 1 www www 663 Apr  3 17:23 bpp_development/exports/coder-table_2020-04-03_172309.csv
```

So that's one in the production deployment created (by you, I think) at 7:13 this evening, as well as one each in the testing and development deployments that I created during the 5:00 hour when I was building the change...  But maybe I'm overlooking another location?

**Comment by davidskalinder** [comment]
As with #66, this seems to be working well for now in two rounds of processing.
